devlab/config.template

```diff
@@ -20,6 +20,9 @@ ssh_chunk_size = 100
 retry = 5
 migrate_extnets = True
 time_wait = 5
+all_vms = False
```

Comment:
I believe it is False in CF direcory: configs/config.ini
So, i've just copied it to template as is.